website/other/bezier-rs-demos/src/App.vue

```diff
@@ -288,6 +288,93 @@ export default defineComponent({
 						},
 					},
 				},
+				{
+					name: "Intersect (Line Segment)",
+					callback: (bezier: WasmBezierInstance): string => {
+						const line = [
+							{ x: 150, y: 150 },
+							{ x: 20, y: 20 },
+						];
+						const mappedLine = line.map((p) => [p.x, p.y]);
```

Comment:
I don't really see the point of this - why can't you just define the line as 
```
const line = [
    [150, 150],
    [20, 20],
];
```
to begin with (here and below)?